src/documents/IoCReferences.md

```diff
@@ -98,7 +98,7 @@ For a conventional IoC reference (of the style `<componentRef>` rather than the
 
 The set of components which are in scope for resolution from this site are shown in yellow in this diagram. These are components which are either i) an ancestor of the component holding the reference site, or ii) a sibling of such a component. The context reference matches a component if it matches via one of the 3 rules in the above table - **either** it agrees with a fully-qualified grade or type name of a component, **or** it agrees with the last path segment of such a name. If no context name matches anywhere in the tree, the reference expression resolves to `undefined`. In this case, if the path segments following the context name in the reference expression are not empty, the framework will throw an error.
 
-## Examples of `{<componentRef>}` ##
+## Examples of {<componentRef>} ##
```

Comment:
This doesn't render properly - the angle brackets and their content are stripped.